### PR TITLE
Remove confirmation of own pending creation

### PR DESCRIPTION
**Pull request by Elweyn**
## 🍰 Pullrequest
A Moderator can prepare some pending creation for his own, but he is not allowed to confirm them!

### Issues
- None

### Todo
- [X] None

### CODE REVIEW

**Review by ogerly — APPROVED** [review]
i confirm the behaviour. 
i have made a creation as admin and cannot confirm it. i get an error message from the backend.

File: backend/src/graphql/resolver/AdminResolver.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export class AdminResolver {
     return await getUserCreations(user.id)
   }
 
-  // @Authorized([RIGHTS.SEARCH_USERS])
+  @Authorized([RIGHTS.SEARCH_USERS])
```
Comment (ulfgebhardt) [inline backend/src/graphql/resolver/AdminResolver.ts:69]: This seems to be the wrong right - every function gets its own right.

File: backend/src/graphql/resolver/AdminResolver.ts [hunk 2]
```diff
@@ -94,24 +94,9 @@ export class AdminResolver {
...
-    //   loginPendingTaskAdmin.date = creationDateObj
-    //   loginPendingTaskAdmin.memo = memo
-    //   loginPendingTaskAdmin.moderator = moderator
-    //
-    //   pendingCreationRepository.save(loginPendingTaskAdmin)
-    // }
-    // return await getUserCreations(user.id)
   }
 
+  @Authorized([RIGHTS.SEARCH_USERS])
```
Comment (ulfgebhardt) [inline backend/src/graphql/resolver/AdminResolver.ts:99]: This seems to be the wrong right - every function gets its own right.

File: backend/src/graphql/resolver/AdminResolver.ts [hunk 3]
```diff
@@ -139,6 +124,7 @@ export class AdminResolver {
     return pendingCreationsPromise.reverse()
   }
 
+  @Authorized([RIGHTS.SEARCH_USERS])
```
Comment (ulfgebhardt) [inline backend/src/graphql/resolver/AdminResolver.ts:127]: This seems to be the wrong right - every function gets its own right.

File: backend/src/graphql/resolver/AdminResolver.ts [hunk 4]
```diff
@@ -147,11 +133,17 @@ export class AdminResolver {
     return !!res
   }
 
+  @Authorized([RIGHTS.SEARCH_USERS])
```
Comment (ulfgebhardt) [inline backend/src/graphql/resolver/AdminResolver.ts:136]: This seems to be the wrong right - every function gets its own right.

**Review by ulfgebhardt — COMMENTED** [review]
Looks good besides the rights thingy

File: backend/src/graphql/resolver/AdminResolver.ts (hunk 1 above)
Comment (Elweyn) [reply backend/src/graphql/resolver/AdminResolver.ts:69]: OK, I was not sure while implementing it, I understood that SEARCH_USERS is the actual Admin role / right

File: backend/src/graphql/resolver/AdminResolver.ts (hunk 2 above)
Comment (Elweyn) [reply backend/src/graphql/resolver/AdminResolver.ts:99]: OK, I was not sure while implementing it, I understood that SEARCH_USERS is the actual Admin role / right

File: backend/src/graphql/resolver/AdminResolver.ts (hunk 3 above)
Comment (Elweyn) [reply backend/src/graphql/resolver/AdminResolver.ts:127]: OK, I was not sure while implementing it, I understood that SEARCH_USERS is the actual Admin role / right

File: backend/src/graphql/resolver/AdminResolver.ts (hunk 4 above)
Comment (Elweyn) [reply backend/src/graphql/resolver/AdminResolver.ts:136]: OK, I was not sure while implementing it, I understood that SEARCH_USERS is the actual Admin role / right

**Pull request merged**